doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -75,6 +225,124 @@ The following UICR fields are supported:
    If no UICR values are programmed, |ISE| applies a set of default configurations.
    Applications that do not require custom settings can rely on these defaults without modifying the UICR.
 
+UICR image generation
+=====================
+
```

Comment:
Should we have one sentence of motivation here?

```
A file containing the UICR contents is needed when non-default configurations are needed for an application.
This file is generated through the ``UICR image`` - which is a zephyr image without firmware, just configurations. 
```

Not very good phrasing from my side, but I guess you get the point.